portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,32 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+var i = 0;
+var txt = 'Hi, my name is Max./Welcome to my portfolio.';
+var speed = 50;
+
+function typewriter() {
+    console.log("test2");
+    if (i < txt.length) {
```

Comment:
Question: why do you use if here instead of while? It seems that this function is only called once (on body load).

Reply:
I used an if because I am recursively calling the function using the setTimeout function. This allows me to make the function run every 50ms and create a typing effect.

Reply:
oh got it. That's a fancy way to call recursive functions